docker-compose-alerts.yml

```diff
@@ -26,7 +26,7 @@ services:
     profiles: ["python"]
     ports:
       # Port upon which to serve webapp
-      - "${WEB_SERVER_PORT:-8080}:80"
+      - "${POST_PORT:-8080}:80"
```

Comment:
I think it would be better it the configuration backend and delivery backend weren't running on the same container/port. They have very different purposes and scaling needs. Without needing to rework a bunch of stuff, you could just define a second docker-compose service which uses the same image but runs it on a different port.

Reply:
Could you explain this a bit further? Currently the port is managing the front end service - would making a new port mean that the Flask app runs on both ports simultaneously, or would it run 2 Flask apps one on each port?